contracts/Oracle/Calculations/Synth.sol

```diff
@@ -0,0 +1,171 @@
+        }
+        for (uint i = 0; i < _chfSynths.length; i++) {
+            chfSynths[_chfSynths[i]] = true;
+        }
+        for (uint i = 0; i < _audSynths.length; i++) {
+            audSynths[_audSynths[i]] = true;
+        }
+        for (uint i = 0; i < _jpySynths.length; i++) {
+            jpySynths[_jpySynths[i]] = true;
+        }
+        for (uint i = 0; i < _krwSynths.length; i++) {
+            krwSynths[_krwSynths[i]] = true;
+        }
+    }
+    
+    function setEurSynth(address synthAddress, bool isSynth) public onlyManagers {
+        eurSynths[synthAddress] = isSynth;
+    }
+    
+    function setGbpSynth(address synthAddress, bool isSynth) public onlyManagers {
+        gbpSynths[synthAddress] = isSynth;
+    }
+    
+    function setChfSynth(address synthAddress, bool isSynth) public onlyManagers {
+        chfSynths[synthAddress] = isSynth;
+    }
+    
+    function setAudSynth(address synthAddress, bool isSynth) public onlyManagers {
+        audSynths[synthAddress] = isSynth;
+    }
+    
+    function setJpySynth(address synthAddress, bool isSynth) public onlyManagers {
+        jpySynths[synthAddress] = isSynth;
+    }
+    
+    function setKrwSynth(address synthAddress, bool isSynth) public onlyManagers {
+        krwSynths[synthAddress] = isSynth;
+    }
+    
+    function getEurPrice() public view returns (int256) {
+        (,int price,,,) = eurChainlinkFeed.latestRoundData();
+        return price;
+    }
+    
+    function getGbpPrice() public view returns (int256) {
+        (,int price,,,) = gbpChainlinkFeed.latestRoundData();
+        return price;
+    }
+    
+    function getChfPrice() public view returns (int256) {
+        (,int price,,,) = chfChainlinkFeed.latestRoundData();
+        return price;
+    }
+    
+    function getAudPrice() public view returns (int256) {
+        (,int price,,,) = audChainlinkFeed.latestRoundData();
+        return price;
+    }
+    
+    function getJpyPrice() public view returns (int256) {
+        (,int price,,,) = jpyChainlinkFeed.latestRoundData();
+        return price;
+    }
+    
+    function getKrwPrice() public view returns (int256) {
+        (,int price,,,) = krwChainlinkFeed.latestRoundData();
+        return price;
+    }
+    
+    function getPriceUsdc(address tokenAddress) public view returns (int256) {
+        int256 price;
+        
+        if (eurSynths[tokenAddress]) {
+            price = getEurPrice();
+        } else if (gbpSynths[tokenAddress]) {
+            price = getGbpPrice();
+        } else if (chfSynths[tokenAddress]) {
+            price = getChfPrice();
+        } else if (audSynths[tokenAddress]) {
+            price = getAudPrice();
+        } else if (jpySynths[tokenAddress]) {
+            price = getJpyPrice();
+        } else if (krwSynths[tokenAddress]) {
+            price = getKrwPrice();
+        } else {
+            revert("token not a synth");
+        }
+        
+        return price / 10 ** 2;
```

Comment:
does 0.8.2 automatically include safeMath? from always working with 0.6.12 I feel triggered seeing naked math like this lol

Reply:
Looks like it does - https://docs.soliditylang.org/en/v0.8.7/control-structures.html?highlight=checked#checked-or-unchecked-arithmetic